src/pages/docs/game/chat.mdx

```diff
@@ -0,0 +1,20 @@
+---
+layout: ../../../layouts/docs.astro
+---
+
+
+# __Chat__ 
+[@veritanuda](https://keybase.io/veritanuda)
+
+In game chat is now possible by simply clicking the `Chat` button in the bottom left of the viewport window.
+
+![Chat button](assets/chat_button_default.png)
+
+When a chat message is sent by any user then a number of unread messages will appear next to the button.
+
+![Chat Button Messages waiting](assets/chat_button_waiting.png)
+
+The chat window accepts Markdown input meaning styles and links can be included as well as more advanced formatting. See the [Markdown Tutorial](../../../learn/dm/Markdown_Tutorial/markdown)
```

Comment:
The chat window also has some custom text features like automatically converting an image url to an image when it appears in chat regardless of the `![]()` syntax.

We could mention this either here or in the markdown guide.

Reply:
Oh I did not know that. Is that true for notes as well?

Reply:
Hmm I just tried that and it does not seem to work. If I add the inline syntax it worked fine? 
![Screenshot from 2024-06-26 17-26-34](https://github.com/Kruptein/planarally-docs/assets/7777993/34b5d0c4-0d0a-42c8-825a-4ebf70882717)

Reply:
Ah well, that happens to be an image hosted on a server with strict CORS settings which disallows manual requests which the fancy functionality relies on.
Most images will be hosted on non-cors routes as it makes little sense, but it will indeed sometimes happen I guess.

Reply:
In which case prolly best not put it in as a default will work then as I can pick random images and a percentage won't work. 

FWIW it does not work with my Nextcloud paths either whereas the `![]()` method does work.

Reply:
Well people can use what they want, it's an option. All the images I've been using in my own campaigns since introducing it have worked fine.

There is no real reason why images are behind a CORS header anyway, so your nextcloud is just overly strictly configured I would say. But feel free to keep using the ![]() syntax.

But it's good to know that CORS messes with it, something I can try to look into in the future.

Reply:
Yeah, it is kinda weird why the full syntax works while the shortcut does not, but either way, I don't feel we should add that shortcut to the documentation if it is going to be so inconsistent.

Reply:
There is nothing weird about it. That's how CORS works. Scripted requests get blocked by the browser if the target source explicitly marks the files as not allowed to fetch. Which is the case for your images.

The vast majority of users will not know about markdown and just paste urls anyway, so I guess it doesn't really matter if you include it or not, but I would rather you did.

As I mentioned it's a misconfiguration to have cors headers on images, but I'll add a fallback mechanism to go through the server instead in the near future.

Reply:
Fairy nuff. I can put a line in about it, but also say it may not always work with all URLs. I don't most people even know what CORS is, let alone how to spot a server that has it or not.